script.js

```diff
@@ -0,0 +1,18 @@
+var url = 'http://api.icndb.com/jokes/random';
+var button = document.getElementById('get-joke');
+button.addEventListener('click', function(){
+  getJoke();
+});
+var paragraph = document.getElementById('joke');
+
+function getJoke() {
+  var xhr = new XMLHttpRequest();
+  xhr.open('GET', url);
+  xhr.addEventListener('load', function(){
+    var response = JSON.parse(xhr.response);
+    paragraph.innerHTML = response.value.joke;
```

Comment:
at first (before you assign to `innerHTML`) you should check that `value` and `joke` exist in your response